### PR TITLE
Read a book endpoint

### PR DESCRIPTION
**Pull request by mattgen88**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,23 +2,20 @@ module github.com/exlibris-fed/exlibris
 
 go 1.14
 
+replace github.com/exlibris-fed/openlibrary-go => /home/mgeneral/repos/openlibrary-go
```
Comment (Sigafoos) [inline go.mod:5]: can you take this out?

File: handler/auth.go [hunk 2]
```diff
@@ -81,6 +81,7 @@ func (h *Handler) Register(w http.ResponseWriter, r *http.Request) {
 		log.Printf("error sending registration email to user %s: %s", user.Username, err.Error())
 	}
 
+	w.Header().Add("Content-Type", "application/json")
```
Comment (Sigafoos) [inline handler/auth.go:84]: But there's no content...?

File: main.go [hunk 3]
```diff
@@ -68,7 +68,7 @@ func main() {
 	api.HandleFunc("/verify/{key}", h.VerifyKey).Methods(http.MethodGet, http.MethodOptions)
 
 	books := api.PathPrefix("/book").Subrouter()
-	books.Use(m.Authenticated)
+	books.Use(m.WithUserModel)
```
Comment (Sigafoos) [inline main.go:71]: Does `WithUserModel` imply `Authenticated`? Otherwise we'd need both.

File: model/author.go [hunk 4]
```diff
@@ -35,3 +36,8 @@ func (a *Author) ToType() vocab.Type {
 
 	return author
 }
+
+// NewAuthor creates an author from an openlibrary Author
+func NewAuthor(author openlibrary.Author) *Author {
+	return &Author{}
```
Comment (Sigafoos) [inline model/author.go:42]: `// TODO` I assume

File: main.go (hunk 3 above)
Comment (mattgen88) [reply main.go:71]: Seems to work?

File: model/author.go (hunk 4 above)
Comment (mattgen88) [reply model/author.go:42]: Yeah

File: handler/auth.go (hunk 2 above)
Comment (mattgen88) [reply handler/auth.go:84]: I got a little happy with the content types, whoops

File: go.mod (hunk 1 above)
Comment (mattgen88) [reply go.mod]: Already fixed

**Pull request merged**